content/docs/lifting-state-up.md

```diff
@@ -24,9 +27,10 @@ function BoilingVerdict(props) {
 }
 ```
```

Comment:
In het voorbeeld zouden ook de afgedrukte teksten kunnen worden vertaald:

Het water zou koken.
Het water zou niet koken.

```
function BoilingVerdict(props) {
  if (props.celcius >= 100) {
    return <p>Het water zou koken.</p>
  }
  return <p>Het water zou niet koken.</p>;
}
```